app/uk/gov/hmrc/gatekeeperemail/models/User.scala

```diff
@@ -16,6 +16,9 @@
 
 package uk.gov.hmrc.gatekeeperemail.models
 
+import scala.collection.JavaConverters.asScalaBufferConverter
```

Comment:
Don't use JavaConverters in a Scala 2.13 project - use scala.jdk.CollectionConverters instead